frontend/src/components/playground/boardItemI18nStrings.tsx

```diff
@@ -0,0 +1,9 @@
+"use client";
+export const boardItemI18nStrings = {
```

Comment:
Nit: also can we associate the type for the boardItemStrings?
```suggestion
export const transformationBoardItemStrings = {
```